ooiui/static/js/views/common/NavbarView.js

```diff
@@ -60,8 +60,10 @@ var NavbarView = Backbone.View.extend({
     // Messages only appear to logged in users
     if(ooi.login.loggedIn()) {
         this.$el.find('#navbar-menus').append(this.templates.logged_in_nav_items());
+        this.$el.find('#Science').hide();
         //this.$el.find('#navbar-menus').append(this.messageView.el);
     }
     this.$el.find('#navbar-menus').append(this.dropdownUserView.el);
+   
   }
-});
```

Comment:
I think Dan is talking about line 63 this.$el.find('#Science').hide();

My answer: I have no idea.

We already have a Logged in partial. So I added the logged in science drop down to the log in partial and hide the other one.  When you log out the logged in partial goes away and you see the original link. I"m not adding a class just hiding it, if a user is logged in.